tests/utils/ssh/ssh_default.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Exposes util to invoke remote commands on non-windows hosts via ssh.
+
+// +build !runoncewin
+
+package ssh
```

Comment:
let's not introduce "_" with this PR ... `tests/*` we have tried to avoid introducing that unless it is necessary.